### DISCUSSION TITLE
New collection: hpe.ilo

### DISCUSSION THREAD

**Discussion by rajeevkallur** [ansible-collections/ansible-inclusion#61] [Invalid]
Ansible Galaxy: https://galaxy.ansible.com/hpe/ilo
GitHub: https://github.com/HewlettPackard/ilo-ansible-collection
Issues Tracker: https://github.com/HewlettPackard/ilo-ansible-collection/issues
GitHub Handles:  @rajeevkallur 
Is the collection part of Automation Hub: Soon
We meet [Collection Requirements](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst): Yes

**Comment by Andersson007** [comment]
@rajeevkallur (sorry, deleted the first comment accidentally) hello, thanks for the inclusion request!
The collection will be reviewed as soon as there are volunteers in the community and steering committee.
The request will be published in [Bullhorn](https://github.com/ansible/community/wiki/News#the-bullhorn) every week (except the next one, will be on PTO).
[Reviewing](https://github.com/ansible-collections/ansible-inclusion#submission-process) other's requests can speed up the process.
Thanks

**Comment by mariolenz** [comment]
@rajeevkallur I don't want to start an official review at the moment (no time!) but as far as I can see, you don't [tag your releases](https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html#repository-management) and I don't see any [CI Testing](https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html#ci-testing).

**Reply by Andersson007** [reply]
@rajeevkallur please go through https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst carefully.
after you fix stuff, you can check it yourself first using this [review checklist](https://github.com/ansible-collections/overview/blob/main/collection_checklist.md). thanks

**Reply by rajeevkallur** [reply]
@mariolenz, Is CI Testing must?

**Reply by mariolenz** [reply]
@rajeevkallur At least for sanity tests I think it is.

Where's the problem? [CI Testing](https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html#ci-testing) says: You can copy the free-to-use [GitHub action workflow file](https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml) from the [Collection Template repository](https://github.com/ansible-collections/collection_template/) to the _.github/workflows_ directory in your collection to set up testing through GitHub actions. So it's really easy to implement.

Anyway, how do you make sure that changes to your collection are OK, or that your collection still works with a new ansible-core release without a CI?

Of course, you can run this daily (or at least once per week) manually, but since this would be a pain in the ass I suggest using some kind of CI pipeline. You don't even have to use GitHub Actions, but it would be great to have access to your pipeline so we see those tests are really running and, above all, succeeding.

**Reply by Andersson007** [reply]
i think reading [Collection Requirements](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst) rather than just copy-pasting them to the description is all what we need here. There's a corresponding section. Thanks @mariolenz for the answer

**Comment by mariolenz** [comment]
The last release of this collection has been more than 1 1/2 years (May 28, 2024), and the [last commit](https://github.com/HewlettPackard/ilo-ansible-collection/commit/b5855a0978195287f1599d45432f0c6fe808320c) 8 months ago . I don't see any tests with the current ansible-core version anywhere, either.

So I suggest to close this or move it to invalid or suspended or whatever.

**Reply by Andersson007** [reply]
@mariolenz done, thank you!
